frontend/src/app/cookie-law-banner/cookie-law-banner.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { CookieService } from 'ngx-cookie';
+
+@Component({
+  selector: 'volontulo-cookie-law-banner',
+  templateUrl: './cookie-law-banner.component.html',
+  styleUrls: ['./cookie-law-banner.component.css']
+})
+export class CookieLawBannerComponent implements OnInit {
+
+  constructor(private _cookieService: CookieService) {
+  }
+
+  ngOnInit() {
+    this.hideCookieBanner(!!this._cookieService.get('cookielaw_accepted'));
+  }
+
+  acceptCookieLaw() {
+    const yearFromNow = new Date().setFullYear(new Date().getFullYear() + 1);
+    this._cookieService.put('cookielaw_accepted', '1', {
+      expires: new Date(yearFromNow)
```

Comment:
do we need to sate expiration date?

Reply:
The django cookie-law plugin set the expiration date to 10 years from now, the default for ngx-cookie was a session cookie. I figured a year would work. You decide.